File: 8kyuTotalPoints.js
//8kyu Total Points
function points(games) {
    let total = 0;
    games.map(game => {
      if (game[0] === game[2]) {
        total += 1;
      } else if (game[0] > game[2]) {
        total += 3;
      }
    });
    return total;
  }
//test points(['1:0','2:0','3:0','4:0','2:1','3:1','4:1','3:2','4:2','4:3'])

File: C#/8kyu/8kyuTotalPoints.cs
using System.Linq;
using System;
public static class Kata {
    public static int TotalPoints(string[] games) {
        int total = 0;
        foreach (string game in games) {
          if (game[0] > game[2])
            total += 3;
          else if (game[0] == game[2])
            total += 1;            
        }
        return total;
    }
}